### PR TITLE
Es6 adapter.js

### PR DESCRIPTION
**Pull request by foxriver76**

### CODE REVIEW

File: packages/adapter/src/lib/adapter/utils.ts [hunk 1]
```diff
@@ -260,4 +260,63 @@ export class Utils {
...
         return result;
     }
+
+    /**
+     * Validates the object-type argument that is passed to setState
+     * @param obj object to validate
+     */
+    validateSetStateObjectArgument(obj: Record<string, any>): void {
+        // Check that we have at least one existing non-undefined property at all, else invalid
+        if (!Object.keys(obj).some(key => obj[key] !== undefined)) {
```
Comment (AlCalzone) [inline packages/adapter/src/lib/adapter/utils.ts:270]:
```suggestion
        if (!Object.values(obj).some(prop => prop !== undefined)) {
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Holy shit, it's happening 🙈

File: packages/adapter/src/lib/adapter/utils.ts (hunk 1 above)
Comment (foxriver76) [reply packages/adapter/src/lib/adapter/utils.ts:270]: > Holy shit, it's happening 🙈

But I think tests are failing now. Have to look into it where I messed it up with receiving messages

**Comment by foxriver76** [comment]
Redis tests are failing

```
    ✔ Tests REDIS test files: getBinaryState
 redis subscribeMessage messagebox.system.adapter.test.0
 States system redis pmessage messagebox.system.adapter.test.0/messagebox.system.adapter.test.0:{"test":1,"_id":56431684}
state changed. messagebox.system.adapter.test.0
    1) Tests REDIS test adapter: check pushMessage
```

seems like message receives but message listener/method not triggered

**Comment by foxriver76** [comment]
strange locally all tests pass

**Pull request merged**